### ISSUE TITLE
Folios F7 

### ISSUE THREAD

**Issue by seesmith** [UoEMainLibrary/LibraryMapping#90]
The Dewey classified books on the 4th floor has a pamphlet (p .?) and  folio (f  .?)section at the start of the collection, before all the other "normal size" book stock.
 
However, Fine Art folios, F .7's, and "normal" .7 books have never been separated from each other, due to the size of "normal" sized books within this collection.
By nature, most Art books tend to be large and heavy, and F .7s and .7's, have always been interfiled.
It is the only section like this.
 
So to sum up:-
F .7's should not show on a map as being "Located" beside all other Folio items within the Folio section, but should be located with all other .7 books on this floor.
Pamphlets P .7 - are with all other pamphlets (in boxes) at the beginning of the book sequence.
